Face/Identify/Program.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.CognitiveServices.Vision.Face;
+using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+
+namespace Identify
+{
+    public class Program
+    {
+        // Create a client.
+        static string ApiKey = "ENTER YOUR KEY HERE";
+
+        static IFaceClient Client = new FaceClient(new ApiKeyServiceClientCredentials(ApiKey))
+        {
+            Endpoint = "ENTER YOUR ENDPOINT HERE"
+        };
+
+        public static List<string> TargetImageFileNames =
+            new List<string> { "Family1-Dad1.jpg", "Family1-Daughter1.jpg", "Family1-Mom1.jpg", "Family1-Son1.jpg", "Family2-Lady1.jpg", "Family2-Man1.jpg", "Family3-Lady1.jpg", "Family3-Man1.jpg" };
+
+        public static string SourceImageFileName = "identification1.jpg";
+
+        public static void Main(string[] args)
+        {
+            Identify_in_PersonGroup().Wait();
+            Identify_in_LargePersonGroup().Wait();
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+
+        public static async Task Identify_in_PersonGroup() 
+        {
+            // Create a person group.
+            string GroupId = Guid.NewGuid().ToString();
+            Console.WriteLine($"Create a person group ({GroupId}) {Environment.NewLine}");
+            await Client.PersonGroup.CreateAsync(GroupId, GroupId);
+
+            foreach (var targetImageFileName in TargetImageFileNames) 
+            { 
+                // Read image file. 
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open)) 
+                {
+                    // Create a person group person.
+                    Person p = new Person();
+                    p.Name = targetImageFileName;
+                    p.UserData = "Person for sample";
+                    Console.WriteLine($"Create a person group person '{p.Name}'");
+                    p.PersonId = (await Client.PersonGroupPerson.CreateAsync(GroupId, p.Name)).PersonId;
+
+                    // Add face to the person group.
+                    Console.WriteLine($"Add faces to the person group from image `{targetImageFileName}`. {Environment.NewLine}");
```

Comment:
`faces` to `face` for clarification.
Please help modify all same issues.

Reply:
done